src/components/MobileMetricCard/MobileMetricCard.module.scss

```diff
@@ -0,0 +1,35 @@
+@import '~@santiment-network/ui/mixins';
+@import '~@santiment-network/ui/variables';
+
+.metric {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 14px 0;
+  border-bottom: 1px solid var(--porcelain);
+
+  &__right {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+  }
+
+  &__name {
+    @include text-normal;
+
+    line-height: 20px;
+    margin: 0;
+  }
+
+  &__value {
+    @include text-normal;
+
+    line-height: 20px;
+  }
+
+  &__changes {
+    @include text-normal;
+
+    line-height: 20px;
+  }
```

Comment:
The repeated code though LOC `17-34`. `h3.metric__name` could be a simple `div` for avoiding specifying `margin` prop. `text-normal` and `line-height` could be inherited from the parent node (might not work. In such case we can just create a single class `&__text`)